code/datums/elements/food/processable.dm

```diff
@@ -12,8 +12,11 @@
 	var/amount_created
 	///Whether or not the atom being processed has to be on a table or tray to process it
 	var/table_required
+	///Verb used in processing food (such as slice, flatten), defaults to process
+	var/screentip_verb
 
-/datum/element/processable/Attach(datum/target, tool_behaviour, result_atom_type, amount_created = 3, time_to_process = 2 SECONDS, table_required = FALSE)
+/datum/element/processable/Attach(datum/target, tool_behaviour, result_atom_type,
+	amount_created = 3, time_to_process = 2 SECONDS, table_required = FALSE, screentip_verb = "Process")
```

Comment:
```suggestion
/datum/element/processable/Attach(datum/target, tool_behaviour, result_atom_type, amount_created = 3, time_to_process = 2 SECONDS, table_required = FALSE, screentip_verb = "Process")
```

it just looks odd otherwise imo